README.md

```diff
@@ -253,18 +253,18 @@ function* treeWalker(refresh) {
           // object.
           defaultHeight: 30,
           id,
-          isLeaf: children.length === 0,
+          isLeaf: children?.length === 0,
           isOpenByDefault: true,
           name,
           nestingLevel,
         }
       : id;
 
-    if (node.children.length !== 0 && isOpened) {
-      for (let i = node.children.length - 1; i >= 0; i--) {
+    if (children?.length !== 0 && isOpened) {
+      for (let i = children?.length - 1; i >= 0; i--) {
```

Comment:
```suggestion
    if (children && children.length !== 0 && isOpened) {
      for (let i = children.length - 1; i >= 0; i--) {
```
I think it would be better to check it this way, because `undefined !== 0` would give us `true`; also I don't see any reason to check the `children` in the `for` loop again.